### ISSUE TITLE
Jump start and use QAN to finetune

### ISSUE THREAD

**Issue by LinZichuan** [bigaidream-projects/qan#24]

**Comment by bigaidream** [comment]
Does not work well.

**Comment by bigaidream** [comment]
After training a network to e.g. 90% accuracy, we save the model. 

Then we tune learning rates from there repeatedly.

**Comment by bigaidream** [comment]
Not that important at this moment.
